modules/client/src/client-common/constants.ts

```diff
@@ -21,6 +21,58 @@ export const UNAVAILABLE_PROPOSAL_METADATA: ProposalMetadata = {
   resources: [],
 };
 
+// TODO change rpcs to aragon ones
+export const WEB3_NODES: { [K in SupportedNetworks]: string[] } = {
+  homestead: ["https://mainnet.eth.aragon.network"],
+  goerli: ["https://endpoints.omniatech.io/v1/eth/goerli/public"],
+  matic: ["https://polygon.llamarpc.com"],
+  maticmum: ["https://endpoints.omniatech.io/v1/matic/mumbai/public"],
+};
+
+const getGraphqlNode = (netowrk: string): string => {
+  return `https://subgraph.satsuma-prod.com/${
+    process.env.SATSUMA_API_KEY || ""
```

Comment:
Okay, I'll create a couple of constants for this, I didn't want to expose the satsuma api keys and ifs nodes keys but if there is no other way :(

Reply:
Not sure what do you mean here. The deveolpers can pass another node if they want, this was supposed to be our api key for satsuma, thinking about it it does not make sense to hide it, since is fairly public, you can find it bu checking the nerwork tab in app.argaon.org